noether_tpp/tool_path_modifiers/include/noether_tpp/tool_path_modifiers/default_modifiers.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <noether_tpp/core/tool_path_modifier.h>
+
+namespace noether_tpp
+{
+/**
+ * @brief Creates a tool path modifier that enforces the definition of an edge path on the
+ * path generated by a EdgePlanner.  It will enforce uniform segment ordering, uniform loop
+ * direction, and ordering of loops by descending loop length.
+ * @return
+ */
+struct DefaultEdgePlannerModifier : public OneTimeToolPathModifier
```

Comment:
I don't like putting adjectives like "default", "simple", etc. in names. It takes attention away from what it actually does.

Reply:
This might be a corner case where the word default is appropriate. The general idea is that raster and edge tool path plans should have various meta-properties (e.g. consistent ordering, consistent orientation; these are documented in the README). These two classes will be implemented to enforce these "default" behaviors for each planner, hence the name default. I wasn't able to think of a better name, but I'm open to suggestions if you can think of something more descriptive

Reply:
We could call them "EnforceEdgePlanDefinition" and "EnforceRasterPlanDefinition", but I honestly think that "default", here, is appropriate.  I agree that "simple", "default", "advanced", "improved", etc., generally make bad names, but here it is actually *implementing the definition of the default behavior*.

Reply:
Saying the modifier enforces a default behavior is subtly different from saying it's the "default" modifier, with the latter not actually explaining anything by name alone. The main alternative I would consider is not having one modifier that does everything (and is difficult to name) but having distinct modifiers with clear separate jobs and having the default pipeline include that set.

Reply:
I think we went this route originally because we thought the planner classes would want/need to store this "default" modifier class as a member, so it made sense to create a single class to house all the modification behavior. As it stands now, these default modifier classes are not a member of the tool path planner classes; they are just constructed and used in the `plan` function. Instead of having a single class to do all of the default behavior modifications, maybe we should just create the individual modifiers that we need directly in the `plan` class and call them in the right order. Then we could get rid of this default modifier class and the issues associated with it